std_msgs/Header header
float64 ship_pitch               # ship_pitch reading
float64 ship_pitch_variance      # ship_pitch_variance reading
